### PR TITLE
Simplify swarm client lib installation

### PR DESCRIPTION
**Pull request by htjain**
Changes:

1. Updated example build task yaml and docker files
   (in case of run-sl exmaples) to use wheel file hosted in github repo
2. Removed temp volume creation section for current wheel file handling.
3. removed soft link of swam lib wheel file

### CODE REVIEW

File: examples/mnist/README.md [hunk 1]
```diff
@@ -103,17 +103,7 @@ The cluster setup for this example uses 2 hosts, as shown in the figure below:
    ```
    Note: If the swarm installation directory is different for both the hosts then user need to manually modify the <PROJECT-MODEL> value from the swarm_mnist_task.yaml file to the path that is common to both the hosts.  
 
-7. *On both host-1 and host-2*:  
-   Create a docker volume and copy SwarmLearning wheel file there
-   ```
-   docker volume rm sl-cli-lib
```
Comment (htjain) [inline examples/mnist/README.md:109]: confirm if removing `sl-cli-lib` causing any error in build task.

Comment (htjain) [reply examples/mnist/README.md:109]: swop automatically creates it if needed

File: examples/reverse-proxy/mnist/run-all [hunk 2]
```diff
@@ -33,15 +33,6 @@ else
     exit 1; 
 fi
```
Comment (girisairam450) [inline examples/reverse-proxy/mnist/run-all:35]: Please run automated scripts of reverse proxy examples and capture the logs. To make sure it is running properly.

File: examples/cifar10/README.md [hunk 3]
```diff
@@ -67,9 +67,8 @@ The cluster setup for this example uses 2 hosts, as shown in the figure below:
    ```
 
 5. *On both host-1 and host-2*:  
-   Copy SwarmLearning wheel file inside build context and build docker image for ML that contains environment to run Swarm training of user models.  
+   Build docker image for ML that contains environment to run Swarm training of user models.
```
[the remark this answers is not in the document]
Comment (htjain) [reply examples/cifar10/README.md:70]: Yes it will work

**Pull request closed without merging**